### ISSUE TITLE
Geologist: .org submission issues

### ISSUE THREAD

**Issue by MaggieCabrera** [Automattic/themes#4812]
This is the .org feedback:

Keyboard Navigation

    Theme authors must provide visual keyboard focus highlighting in navigation menus and for form fields, submit buttons, and text links.
    Some resources to help you: Information on the requirement
    Video help for Modals, Links, Forms, Menus
    Check TwentyTwenty using tab and shift+tab as an example 

Mobile menu is not accessible using keyboard navigation.
Extras

    Tested up to: 5.7 test your theme with latest WP version. 

Prefixing

    Provide a unique prefix for everything the Theme defines in the public namespace, including options, functions, global variables, constants, post meta, wp_enqueue_script/style handle names, add_image_size names, wp_script_add_data keys, slugs/ids for new categories created with register_block_pattern_category etc. Theme nav menu locations and sidebar IDs are exceptions. 

Fix the issues and resubmit your theme again.

**Comment by MaggieCabrera** [comment]
The trac ticket: https://themes.trac.wordpress.org/ticket/106674

**Comment by scruffian** [comment]
Did these come up with Quadrat? Should we make the fixes there anyway?

**Comment by MaggieCabrera** [comment]
> Did these come up with Quadrat? Should we make the fixes there anyway?

I don't think they came for Quadrat, no. And some, like the keyboard navigation, are not even directly related to the child, are they? I'm not sure what we should do here.

**Comment by scruffian** [comment]
Fix them in Blockbase if we can I suppose.

**Comment by pbking** [comment]
I went through Geologist (and blockbase) and found that I was able to navigation with the keyboard (both in desktop and responsive sizes) as well as I had expected.  I'm not sure what else is expected there.  There certainly don't seem to be any restrictions in Geologist that aren't in Blockbase, so I agree if something is found it should be fixed there.  I just don't know what that would be.  I didn't test deeply in all browsers (only chrome), but I have validated keyboard usage for Blockbase in Chrome Firefox and Edge prior.

**Comment by pbking** [comment]
With the three (very minor) PR's noted above I think I have fixed everything addressable from the TRAC ticket.  With those changes I believe we can/should resubmit this.

**Comment by jeffikus** [comment]
@pbking it looks like our TRAC ticket has been closed, would we need to resubmit a new ticket?

**Comment by MaggieCabrera** [comment]
New trac ticket: https://themes.trac.wordpress.org/ticket/107158

**Comment by MaggieCabrera** [comment]
I submitted that one but I think we should submit blockbase at least too, since we did the index.php change there...

**Comment by pbking** [comment]
> [Looks good. Schedule it for the next release.](https://themes.trac.wordpress.org/ticket/107158#comment:3)

**Comment by metabreakr** [comment]
In Blockbase (and themes like Geologist), the navigation shows a focus in Safari 15 when clicked on: 

![ucgfxy](https://user-images.githubusercontent.com/39252765/150026752-c465d69d-cd92-4880-b50b-02b008c6bbdf.png)

Is there a way for users to turn this off? Reference: https://wordpress.org/support/topic/menu-borked-on-osx-safari/

EDIT: I could only find this issue in Github, though opened this one just in case: https://github.com/Automattic/wp-calypso/issues/60224
